### ISSUE TITLE
Mid fight masses(Not lite now)

### ISSUE THREAD

**Issue by Rs232k** [luckydog7/Funkin-android#309]
![Screenshot_20210412-232631~01](https://user-images.githubusercontent.com/14124680/114435999-a81a2080-9be6-11eb-9f9f-a7e6ec0c1306.jpg)

!No effects and cutscenes!
Full 4 songs
Added new week
(Maybe bugs)
Download link only in telegram
[Download v2](https://t.me/FNF_Lite/35)

**Comment by BoyfriendFNF** [comment]
can you add a mediafıre link

**Comment by Rs232k** [comment]
> can you add a mediafıre link

Sorry, it's against rules.

**Comment by netonero** [comment]
> > can you add a mediafıre link
> 
> Sorry, it's against rules.

Then whats the point

**Comment by BoyfriendFNF** [comment]
> > can you add a mediafıre link
> 
> Sorry, it's against rules.

????

**Comment by BoyfriendFNF** [comment]
> > can you add a mediafıre link
> 
> Sorry, it's against rules.

wdym

**Comment by netonero** [comment]
Yeah how is it against rules its just sending an alternate link for non-telegram users

**Comment by Rs232k** [comment]
![Screenshot_20210413-015855~01](https://user-images.githubusercontent.com/14124680/114454053-fd146180-9bfb-11eb-87db-ba98186875cc.jpg)

**Comment by DanisLixo** [comment]
yoooooo, intelligence

**Comment by DanisLixo** [comment]
crazy, I'm going to download this shit, I'm going to get a Russian virus

**Comment by DanisLixo** [comment]
ok, i will download russians virus and create a lite version cuz im super duper guy who has nothing to do with his life

**Comment by netonero** [comment]
Thats one stricted ass rules for just one apk

**Comment by DanisLixo** [comment]
> Thats one stricted ass rules for just one apk

what r u talkin about? it s fair

**Comment by netonero** [comment]
> > Thats one stricted ass rules for just one apk
> 
> what r u talkin about? it s fair

True

**Comment by FridayNightSans** [comment]
Then why u post this when you doesnt give we link?

**Comment by Rs232k** [comment]
> Then why u post this when you doesnt give we link?

I've given link.

**Comment by ElPeppyWall** [comment]
> > Then why u post this when you doesnt give we link?
> 
> I've given link.

you have discord or something like that, i want that mod but i don't have telegram

**Comment by DanisLixo** [comment]
making a telegram account is easy, just a phone number and GG. now, if you don’t have a phone number, honestly I don’t know how you’re still alive

**Comment by ElPeppyWall** [comment]
> making a telegram account is easy, just a phone number and GG. now, if you don’t have a phone number, honestly I don’t know how you’re still alive

I already downloaded it and it's epic

**Comment by briannewmc22a** [comment]
Aw man i dont have telegram :(

**Comment by Zizoplays1** [comment]
How did you modded all of this, can you make a video about modding fnf for just newbie modders?

**Comment by Rs232k** [comment]
> How did you modded all of this, can you make a video about modding fnf for just newbie modders?

Modded with code edit and compile. See YouTube guides.

**Comment by Zizoplays1** [comment]
Ok

**Comment by Ruv-lol** [comment]
Ayo for some reason every time I wanna change my character at the debug menu during a game it always crash can u fix it?

**Comment by Rs232k** [comment]
> Ayo for some reason every time I wanna change my character at the debug menu, the game crashes can u fix it?

New version issue, wait week7, i will fix if week7 not fixes this.

**Comment by BoyfriendFNF** [comment]
> making a telegram account is easy, just a phone number and GG. now, if you don’t have a phone number, honestly I don’t know how you’re still alive

me without a phone number: ._.

**Comment by Rs232k** [comment]
Mediafire link added...

**Comment by DanisLixo** [comment]
@Rs232k Lite version is out #313. your welcome :)

**Comment by Justine101213k** [comment]
Hey Why Sarvent Assets And Ruv Assets Is sucks And error LOL Error Assets When The Long notes U will see Sarvent assets and Ruv assets Stop working LOL

**Comment by Justine101213k** [comment]
How did make this mod

**Comment by Rs232k** [comment]
> Hey Why Sarvent Assets And Ruv Assets Is sucks And error LOL Error Assets When The Long notes U will see Sarvent assets and Ruv assets Stop working LOL

I do not know what you are talking about, download a video of my mod with these problems.

**Comment by gameegartxd** [comment]
How do you add week?

**Comment by Benjamin-Link** [comment]
> Хорошо, я скачаю вирус русских и создам облегченную версию, потому что я супер пупер парень, который не имеет ничего общего со своей жизнью

Россия
